### PR TITLE
Ir trades

### PR DESCRIPTION
**Pull request by kzbikowski**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,6 +22,7 @@ META-INF/
 
 # Misc.
 .DS_Store
+xchange-examples/src/main/java/com/xeiam/xchange/examples/independentreserve/IndependentReserveDemoUtils.java
```
Comment (timmolter) [inline .gitignore]: can you remove this?

File: xchange-core/src/main/java/com/xeiam/xchange/dto/Order.java [hunk 2]
```diff
@@ -16,7 +16,7 @@
      * Buying order (you're making an offer)
      */
     BID,
-    /**
+      type, /**
```
Comment (timmolter) [inline xchange-core/src/main/java/com/xeiam/xchange/dto/Order.java]: type?

File: .gitignore (hunk 1 above)
Comment (kzbikowski) [inline .gitignore]: of course, I incorrectly assumed that gitignore is also ignored

File: xchange-core/src/main/java/com/xeiam/xchange/dto/Order.java (hunk 2 above)
Comment (kzbikowski) [inline xchange-core/src/main/java/com/xeiam/xchange/dto/Order.java]: IDE added, removing

**Pull request merged**